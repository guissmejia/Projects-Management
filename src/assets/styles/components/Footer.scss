.Footer {
  width: 100%;
  height: 60px;
  display: flex;
  justify-content: center;
  align-items: center;
  background: #fff;
  p {
    text-align: center;
  }
}