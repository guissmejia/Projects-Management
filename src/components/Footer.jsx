import React from 'react'
import '../assets/styles/components/Footer.scss'

const Footer = () => {
  return (
    <footer className="Footer">
      <p>Made for Guiselle Mej√≠a 2020</p>
    </footer>
  );
};

export default Footer