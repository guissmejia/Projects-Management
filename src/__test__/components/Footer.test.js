import React from 'react';
import { mount } from 'enzyme'
import { create } from 'react-test-renderer'
import Footer from '../../components/Footer';

describe('<Footer/>', () => {
  const footer = mount(<Footer/>);
  test('Render Footer Component', () => {
    expect(footer.length).toEqual(1);
  });

  test('Footer haves 1 paragraph', () => {
    expect(footer.find('p')).toHaveLength(1);
  });

  test('Footer Snapshot', () => {
    const footer = create(<Footer/>);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});