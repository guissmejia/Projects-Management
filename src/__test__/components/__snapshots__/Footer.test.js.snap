// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Footer Snapshot 1`] = `
<footer
  className="Footer"
>
  <p>
    Made for Guiselle Mej√≠a 2020
  </p>
</footer>
`;
